tmt/utils.py

```diff
@@ -2906,6 +2917,14 @@ class ListFormat(enum.Enum):
 _FORMAT_VALUE_LIST_ENTRY_INDENT = '  - '
 
 
+def assert_window_size(window_size: Optional[int]) -> None:
+    if window_size is None or window_size > 0:
+        return
+
+    raise GeneralError(
+        f"Allowed widht of terminal exhausted, output cannot fit into {OUTPUT_WIDTH} columns.")
```

Comment:
Just a small typo here:
```suggestion
        f"Allowed width of terminal exhausted, output cannot fit into {OUTPUT_WIDTH} columns.")
```

Reply:
Fixed in e2d256f8